stable/victoria-metrics/templates/ingress.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "victoria-metrics.fullname" . -}}
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
```

Comment:
General practice is to use template directly:

```
name: {{ template "victoria-metrics.fullname" . }}
```

This eliminates the need to define `$fullName` explicitly.